### PR TITLE
Add wttr API to Weather

### PR DESCRIPTION
**Pull request by turn-a-round**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by turn-a-round** [comment]
Updated with new changes in the source.

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Wow, this is nice 😄 

Thanks for your contribution @turn-a-round.

Can you resolve this review, please?

**Comment by matheusfelipeog** [comment]
When finished, [squash](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all changes into a single commit, please 🙏

File: README.md [hunk 1]
```diff
@@ -1186,6 +1186,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Visual Crossing](https://www.visualcrossing.com/weather-api) | Global historical and weather forecast data | `apiKey` | Yes | Yes |
 | [weather-api](https://github.com/robertoduessmann/weather-api) | A RESTful free API to check the weather | No | Yes | No |
 | [Weatherbit](https://www.weatherbit.io/api) | Weather | `apiKey` | Yes | Unknown |
+| [wttr](https://wttr.in) | :partly_sunny: The right way to check the weather [GitHub](https://github.com/chubin/wttr.in) | No | Yes | Unknown |
```
[the remark this answers is not in the document]
Comment (turn-a-round) [reply README.md]: Done :+1:

**Comment by turn-a-round** [comment]
> When finished, [squash](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all changes into a single commit, please

Can't squash as I've merged with upsteam commits in between. Should I have to Close this PR & raise another?

**Pull request closed without merging**